### DISCUSSION TITLE
wubi (large)按kvfb打不出“哪”字

### DISCUSSION THREAD

**Discussion by SolidBlock-cn** [fcitx5-android/fcitx5-android#367] [Q&A | 问答]
我在小企鹅输入法内安装了Wubi (Large)，发现kvfb打不出“哪“字，只能按kvf打出来。内置的非large的五笔输入法无此问题。

**Comment by rocka** [comment]
由于人手不足，现在的码表输入法确实有些疏于维护 ... 可以提交 PR 到这个仓库：

https://github.com/fcitx/fcitx5-table-extra

**Comment by SolidBlock-cn** [comment]
又发现一个类似问题
xdt可以打出“绑”
xdtb只能打出繁体“綁”，不能打出简体“绑”

**Comment by SolidBlock-cn** [comment]
在Wubi (Large)中，dghg无法输入“古”字

**Comment by yanhuacuo** [comment]
有没有可能，因为码表体量过大，而采取了【出简不出全】的制表方式？
并非错误。

**Reply by SolidBlock-cn** [reply]
我不太了解出简不出全的制表方式。但我用过的其他输入法都没有这种情况。
如果说“古”字不编dghg是为了避免与“䃁”字重码的话，“哪”按kvfb不与任何字重码，但无法用kvfb打不出“哪”字。
而普通的五笔（非large）则没有这种情况。

**Comment by SolidBlock-cn** [comment]
pdsk打不出“寄”字

**Reply by yanhuacuo** [reply]
我看了一下，Wubi (Large) 可能不太适合你，它是【出简不出全】的码表。并不是码表错了。
建议使用其它普通五笔码表，比如 fcitx5 自带的码表。
Wubi (Large) 是面向专业文字处理的超集码表。
